### ISSUE TITLE
Add a rubric entry for the IP licensing restrictions of the method

### ISSUE THREAD

**Issue by jyasskin** [w3c/did-rubric#65]
This might belong under the https://w3c.github.io/did-rubric/#alternatives section since a patent on a method could make it illegal to write an alternate implementation in less than ~20 years.

**Comment by kdenhartog** [comment]
@jandrieu this seems like a good point of criteria that should be added. I think it could go further in the direction of the implementation evaluations as well (this is something that we've been considering when evaluating methods we use) to evaluate the license of the implementation as well.

E.g. if a did method only has a single implementation, which is patented, and the license requires payment to use that implementation I wouldn't want to advocate for it's usage no matter if it's been standardized or not and a rubric evaluation should point this out.

**Comment by jandrieu** [comment]
Yep. I like this. It gets interesting when we distinguish between methods and implementations. 

Do we know of any methods that have licensing separate from any implementation?

**Comment by jyasskin** [comment]
When the methods are developed outside of an organization with a good patent policy, you wouldn't necessarily know about patents covering them before the methods are widely deployed. E.g. https://www.wired.com/1998/11/patent-may-threaten-e-privacy/ was an attempt to do this before the W3C had a strong patent policy.

**Comment by kdenhartog** [comment]
> Do we know of any methods that have licensing separate from any implementation?

I've not heard of any up to this point from what I've seen, but I've not been through every did method. `did:id` may be an example of one that could head in that direction based on what I've understood it's doing at an architectural level and `did:ccp` looks like it was heading that direction from the SDK documentation but it looks like a lot of their code that was available has been taken down now as well.

**Comment by TallTed** [comment]
The past few comments here make me think that an idle thought I discarded earlier today should be rekindled.  To wit --

Perhaps DID implementations should be registered alongside DID methods, which should not list their implementations, as I think it makes more sense for the DID implementations to identify the DID method(s) they implement. 

. o O ( Is it possible to implement/support multiple DID methods with/in a single DID implementation? )

**Comment by jandrieu** [comment]
@jyasskin Indeed. I would expect did:hedera or did:hashgraph to have some patent encumbrances. That's a good point.

**Comment by kdenhartog** [comment]
I know the ledgers do have that issue, but it looked like the SDKs does have a permissive license on it
